src/main/java/com/weaponanimationreplacer/WeaponAnimationReplacerConfig.java

```diff
@@ -11,9 +11,9 @@ public interface WeaponAnimationReplacerConfig extends Config
 		keyName = "showUnequippableItems",
 		name = "Search all items",
 		description = "Include all items when searching for model swaps, which may include new items that runelite does not yet " +
-			"know are equippable, as well as items that are not equippable but do have player models. These items do" +
+			"know are equippable,<br/> as well as items that are not equippable but do have player models.<br/> These items do" +
 			"not have equip slot information, so you will have to right-click the item in the search interface and" +
-			"choose an equip slot. WARNING: This also includes items with no player model."
+			"choose an equip slot.<br/> WARNING: This also includes items with no player model."
```

Comment:
I can also apply this code-readable version if you care which will also fix this 
![image](https://github.com/geheur/weapon-animation-replacer/assets/41973452/f5d05cd9-02d2-4385-9c91-80da73fbf801)

```suggestion
        description = "Include all items when searching for model swaps, " +
            "which may include new items that runelite does not yet know are equippable,<br/> " +
            "as well as items that are not equippable but do have player models.<br/>" +
            "These items do not have equip slot information, so you will have to right-click the item in the search interface and" +
            "choose an equip slot.<br/> WARNING: This also includes items with no player model."
```